webapp/store/reducers/nav.js

```diff
@@ -10,14 +11,21 @@ const navStore = (state = initialState, action) => {
 
   switch (type) {
     case ADD_SIDE_NAV: {
-      const duplicates = newState.sidebar.some(
+      const sidebar = [...newState.sidebar];
+      const duplicates = sidebar.some(
```

Comment:
Can you add a comment here? When creating larger boolean expressions, they can become hard to read and also have unexpected bugs

Reply:
Done